google-cloud-tools-plugin/resources/messages/CloudToolsBundle.properties

```diff
@@ -139,20 +140,20 @@ appengine.stop.modules.version.success.dialog.message=Application has been succe
 appengine.stop.modules.version.success.dialog.title=Application Stopped
 appengine.stop.modules.version.canceled.message=Undeploy Canceled
 appengine.deployment.parameters.panel.title=Deployment Parameters
+appengine.deployment.flex.config.panel.title=App Engine Flexible Configuration
+appengine.deployment.flex.appyaml.label=app.yaml:
 appengine.deployment.error.during.staging=Deployment failed due to staging failing with exit code: {0}.
 appengine.deployment.exception.during.staging=Deployment failed due to an exception while staging the project.
 appengine.deployment.error.staging.yaml=The specified app.yaml configuration file does not exist or is not a valid file.
 appengine.deployment.error.staging.dockerfile=There is no Dockerfile in specified directory.
-appengine.deployment.error.staging.directory.gotosettings=Set a valid Docker directory in Module Settings or use another one.
-appengine.deployment.error.staging.file.gotosettings=Set a valid file for app.yaml in Module Settings or use another one.
 appengine.deployment.error.not.logged.in=You must be logged in to perform this action.
 appengine.deployment.error.suggestflexfacet=If you can't find your artifacts in the deployment run configuration, consider adding Google App Engine Flexible support to your module.
 appengine.deployment.source.not.found.error=Deployment source not found: {0}.
 appengine.deployment.invalid.source.error=Invalid deployment source.
 appengine.deployment.status.deploying=Deploying to App Engine
 appengine.deployment.error.invalid.cloudsdk=Invalid Cloud SDK directory path.
 appengine.deployment.deployallconfigs.checkbox.label=Update dos, dispatch, cron, queues, and datastore indexes
-appengine.deployment.flexible.nogaesupport=No modules with Google App Engine support found. Please provide your configuration files
+appengine.deployment.flexible.nogaesupport=No modules with Google App Engine flexible support found.
```

Comment:
yea its caused by no modules / no modules with flex facet.

I hesitated to mention the facet since the way users are typically expected to add facets is via framework support.

Maybe I should mention framework support more explicitly:
No modules with Google App Engine **framework** support found.